apps/codeforafrica/content/team/jacobo-ottaviani.md

```diff
@@ -0,0 +1,20 @@
+---
+id: 24c487bd-ba84-42df-825e-b11a532786ad
+name: Jacobo Ottaviani
+title: "#### Chief Data Officer"
+description: "**Lorem ipsum dolor sit amet consectetur adipiscing, elit ornare
+  et rhoncus sem, laoreet tortor tristique cursus ultrices. Sed urna nec
+  convallis in tempus lacinia velit viverra neque, molestie malesuada fringilla
+  ut vulputate faucibus ad potenti, nisi enim nam montes nostra curae ridiculus
+  curabitur. Elementum phasellus conubia eget ac dignissim hendrerit ullamcorper
+  magna pretium pulvinar, auctor dui pelle.**"
+image: https://res.cloudinary.com/code-for-africa/image/upload/v1655127335/codeforafrica/images/team/image_11_ch6dnb.jpg
+projects:
+  - 14fdf60f-2928-4776-b208-6641e8413f74
+  - afcfdec9-99d7-4f2c-8bdb-964159c0676d
+  - 07f3259e-2379-4850-83b8-85cacde0fcb8
```

Comment:
Currently not. 
I think defining team members in a Project is the better approach since a project has multiple members and thus we won't need to duplicate projects for different team members.